### ISSUE TITLE
Update library

### ISSUE THREAD

**Issue by And96** [sur950/any_link_preview#61]
Please update http library to latest version 1.1.0 actually

This will solve incompatibility issues with others libraries
```
Because aggregator_news depends on any_link_preview ^3.0.1 which depends on http ^1.0.0, http ^1.0.0 is required.
So, because aggregator_news depends on http ^0.13.6, version solving failed.
```

**Comment by sur950** [comment]
This will be available in Version 3.0.2

**Comment by sur950** [comment]
[any_link_preview: ^3.0.2](https://pub.dev/packages/any_link_preview) is Live now.

Feel free to open this ticket if the issue persists. Thank you!
